client/src/components/BookingsDatePicker/BookingsDatePicker.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable prettier/prettier */
```

Comment:
We should not disable Prettier in our files, please use the formatter for project consistency. This is not good practice.

Reply:
@moffatethan I disabled eslint because it marks errors even if there aren't. Can you help to configure this?